Minio/ApiEndpoints/BucketOperations.cs

```diff
@@ -35,6 +35,101 @@ namespace Minio
 {
     public partial class MinioClient : IBucketOperations
     {
+        /// <summary>
+        /// Check if a private bucket with the given name exists.
+        /// </summary>
+        /// <param name="args">BucketExistsArgs Arguments Object which has bucket identifier information - bucket name, region</param>
+        /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
+        /// <returns> Task </returns>
+        public async Task<bool> BucketExistsAsync(BucketExistsArgs args, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
the signatures of the API operations using args style needs to be defined in IBucketOperations.cs. The deprecated methods such as MakeBucketAsync, RemoveBucketAsync and BucketExistsAsync should internally just call the method using args style so that functional tests run it. The previous call signature needs to be marked as deprecated for these 3 API calls.

Reply:
Marked all the other calls Obsolete with a suitable comment, same as MinioClient constructor.